files/ids/bgee/ea.htm

```diff
@@ -37,12 +37,12 @@
 <br />
 <div class="idsHeader"><a name="6">6 CHARMED</a></div>
 <div class="indent1">
-Creatures under control of the player by the charm spell.
+Uncontrolled ally (green selection circle) of the player.
```

Comment:
Are there any examples of this?

Reply:
> Are there any examples of this?

EE Shaman's summons (even though their uncontrolled behavior is achieved through opcode #365...)

Reply:
Weird stuff, you'd expect the exception to get a new value, not everyone else.